prac_06/guitar_test.py

```diff
@@ -0,0 +1,17 @@
+from prac_06.guitar import Guitar
+
+name = "Gibson L-5 CES"
+year = 1922
+cost = 16035.40
+guitar = Guitar(name, year, cost)
```

Comment:
Why not use the Guitar class directly instead of defining name, year and cost again?